salt/utils/schedule.py

```diff
@@ -889,6 +889,7 @@ def handle_func(self, multiprocessing_enabled, func, data):
                     for global_key, value in six.iteritems(func_globals):
                         self.functions[mod_name].__globals__[global_key] = value
 
+            self.functions.pack['__context__']['retcode'] = 0
```

Comment:
So it will always be `0`?

Reply:
No exactly, `self.functions.pack['__context__']['retcode']` will get updated by `self.functions[func](*args, **kwargs)` execution placed on the next line of code. In case the function called does not provide any `retcode`, then it would be `0` by default. Here I mimic the behavior already existing here:  https://github.com/saltstack/salt/blob/2017.7/salt/minion.py#L1474